server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -862,51 +862,77 @@ else if (platform.contains("osx")) {
         mergedContextWithPodsForObjC.put("systemIncludes", new ArrayList<String>());
         mergedContextWithPodsForObjCpp.put("systemIncludes", new ArrayList<String>());
         mergedContextWithPodsForSwift.put("systemIncludes", new ArrayList<String>());
-
+        // also clearing systemIncludes for c
+        // see https://github.com/defold/extender/issues/389
+        mergedContextWithPodsForC.put("systemIncludes", new ArrayList<String>());
 
         List<String> objs = new ArrayList<>();
 
         if (!pod.swiftSourceFiles.isEmpty()) {
+            // Add -enable-experimental-feature AccessLevelOnImport
```

Comment:
Like the comment says, an improvement for the future would be to add swiftFlags to ext.manifest so that additional swift flags can be added per extension.